src/js/plugins/plugin.autoplay.js

```diff
@@ -0,0 +1,130 @@
+/*global BookReader */
+
+/**
+ * Plugin which adds an autoplay feature. Useful for kiosk situations.
+ */
+jQuery.extend(BookReader.defaultOptions, {
+  enableAutoPlayPlugin: true
+});
+
+/**
+ * @augments BookReader.setup
+ */
+BookReader.prototype.setup = (function(super_) {
+  return function (options) {
+    super_.call(this, options);
+
+    this.auto      = false;
+    this.autoTimer = null;
+    this.flipDelay = 5000;
+  };
+})(BookReader.prototype.setup);
+
+/**
+ * @augments BookReader.init
+ */
+BookReader.prototype.init = (function(super_) {
+  return function (options) {
+    super_.call(this, options);
+
+    if (!this.options.enableAutoPlayPlugin) return;
+    this.bind(BookReader.eventNames.stop, (e, br) => br.autoStop());
+  };
+})(BookReader.prototype.init);
+
+/**
+ * @augments BookReader.bindNavigationHandlers
+ */
+BookReader.prototype.bindNavigationHandlers = (function(super_) {
+  return function() {
+    super_.call(this);
+
+    if (!this.options.enableAutoPlayPlugin) return;
+
+    var self = this;
+
+    // Note the mobile plugin attaches itself to body, so we need to select outside
+    var jIcons = this.$('.BRicon').add('.BRmobileMenu .BRicon');
```

Comment:
Oh, let's also use const/let; I think that should be an easy one to do as well
```suggestion
    const jIcons = this.$('.BRicon').add('.BRmobileMenu .BRicon');
```